types/math.go

```diff
@@ -1,41 +1,9 @@
 package types
 
-import (
-	sdkmath "cosmossdk.io/math"
-)
-
-// Type aliases to the SDK's math sub-module
-//
-// Deprecated: Functionality of this package has been moved to it's own module:
-// cosmossdk.io/math
-//
-// Please use the above module instead of this package.
-type (
-	Int = sdkmath.Int
-)
-
-var (
-	NewInt  = sdkmath.NewInt
-	ZeroInt = sdkmath.ZeroInt
-)
-
 func (ip IntProto) String() string {
```

Comment:
We could remove these if this is ACK: https://github.com/cosmos/cosmos-sdk/issues/15949
But this is out of scope of the https://github.com/cosmos/cosmos-sdk/issues/14405

Reply:
lets do this in a folllow up